performance-tests/bench/bench_exe.mpb

```diff
@@ -1,4 +1,4 @@
-project: dcps_rtps_udp, dcps_rapidjson {
+project: dcps_rtps_udp {
```

Comment:
Does this project, or some related project, need to inherit from `rapidjson`?

Reply:
Yeah, how does this work? Does the core library / ValueReader now require RapidJSON or do we need to specify it here?